react-components/src/components/NodeCacheProvider/types.ts

```diff
@@ -59,6 +60,7 @@ export type Image360AnnotationModel = AnnotationModel & {
 export type Image360AnnotationAssetInfo = {
   asset: Asset;
   assetAnnotationImage360Info: AssetAnnotationImage360Info;
+  positions: Vector3[];
 };
```

Comment:
I wonder if we should just return the `Image360AnnotationObject` instead of the annotation info and the centers

Reply:
Actually - we will need the `AssetAnnotationImage360Info` :thinking: Perhaps this is okay - but it feels like we might as well give the user the option to use `Image360Annotation` too